goldenmaster/examples/olinkserver/main.cpp

```diff
@@ -46,56 +63,56 @@ int main(){
     ApiGear::ObjectLink::RemoteRegistry registry;
     ApiGear::ObjectLink::OLinkHost server(registry);
     server.listen("localhost", 8182);
-    testbed2::ManyParamInterface testbed2ManyParamInterface;
-    auto testbed2OlinkManyParamInterfaceService = std::make_shared<testbed2::OLinkManyParamInterfaceAdapter>(registry, &testbed2ManyParamInterface);
+    auto testbed2ManyParamInterface = std::make_shared<testbed2::ManyParamInterface>();testbed2::ManyParamInterfaceTraced testbed2ManyParamInterfaceTraced(testbed2ManyParamInterface );

```

Comment:
are these on the same line, or does it looks just so in the review view.

Reply:
fixing